### DISCUSSION TITLE
Audio Scene/Source Filters not showing in Touch Portal when VST's are used?

### DISCUSSION THREAD

**Discussion by djbauer69** [Palakis/obs-websocket#1098] [Q&A]
Hi! 

As the title says, When I'm using the TDR VOS SLICKEQ, filters just don't show in Touch Portal when trying to create an action.
But when I remove the VST and reopen TP and OBS, the filters show up. 
The Dev of Touch Portal said they've seen it multiple times and that it's an issue with OBS/Websocket.
Is there any way to resolve this? 

Cheers!

**Comment by tt2468** [comment]
I do not see how this could be a bug with obs-websocket. VST filters are just filters, and have no special handling within OBS or the plugin.

If the touch portal developers really believe that it's a bug with obs-websocket, they should file a bug report with evidence.
